core/dbt/graph/selector.py

```diff
@@ -219,6 +239,27 @@ def search(self, included_nodes, selector):
                 yield node
 
 
+class PathSelector(ManifestSelector):
+    FILTER = SELECTOR_FILTERS.PATH
+
+    def search(self, included_nodes, selector):
+        """Yield all nodes in the graph that match the given path.
+
+        :param str selector: The path selector
+        """
+        # use '.' and not 'root' for easy comparison
+        root = Path.cwd()
+        paths = set(p.relative_to(root) for p in root.glob(selector))
+        search = chain(self.parsed_nodes(included_nodes),
+                       self.source_nodes(included_nodes))
+        for node, real_node in search:
+            if (
```

Comment:
What do you think about doing a prefix match instead of an exact filepath match? That would let us do something like
```
dbt run -m models/path/to/models
```

In this scenario, dbt would select all of the nodes located at or below that specified filepath. You buy it?

Reply:
Sure, I don't have strong feelings about that. I guess it would give us parity with the fqn selector, which seems good.